src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const totalP = document.querySelector('.total-population');
+const averageP = document.querySelector('.average-population');
```

Comment:
Ensure that the elements with classes 'total-population' and 'average-population' exist in the DOM to avoid potential null reference errors.